scripts/schema-manager/manifest-loader.ts

```diff
@@ -0,0 +1,105 @@
+import { TigrisSchema } from '@tigrisdata/core/dist/types';
+import path from 'path';
+import fs from 'fs';
+import * as Log from './logger';
+
+// TODO: allow user to override this
+const SCHEMAS_ROOT = 'lib/models';
+const DB_DIR_SUFFIX = '-db';
+const COLL_FILE_SUFFIX = '-schema.ts';
```

Comment:
Any better suggestions to identify DB directories and schema files? (even if there is no schema defined, we will end up creating database for the user)